### DISCUSSION TITLE
65m objects one-node / one-drive

### DISCUSSION THREAD

**Discussion by kitthaworn-T** [minio/minio#17911] [Q&A] [answered]
I have a question and need assistance. I'm implementing Minio on my Ubuntu server and mounting the path to /minio. Everything works, but when I check the Minio console, I see that the bucket size is shown as approximately 100 GB. However, when I SSH to the Ubuntu server and use the df -h command, it shows that 400 GB is being used.

I don't understand why and how to reduc size on disk
minio version RELEASE.2023-06-02T23-17-26Z 
Runtime: go1.19.9 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2023 MinIO, Inc.

**Comment by klauspost** [comment, marked as answer]
File systems have overhead. With so many small objects, the block size, typically 4K, will be significant.
